imports/plugins/core/navigation/client/components/NavigationTreeContainer.js

```diff
@@ -4,7 +4,8 @@ import IconButton from "@material-ui/core/IconButton";
 import withStyles from "@material-ui/core/styles/withStyles";
 import PencilIcon from "mdi-material-ui/Pencil";
 import CloseIcon from "mdi-material-ui/Close";
-import { SortableTreeWithoutDndContext as SortableTree, removeNodeAtPath } from "react-sortable-tree";
+import SortableTree, { removeNodeAtPath } from "react-sortable-tree";
+// import { SortableTreeWithoutDndContext as SortableTree, removeNodeAtPath } from "react-sortable-tree";
```

Comment:
Please remove commented line.

Reply:
Ok, I'm going to extract the last branch and remove the comment